src/system.cpp

```diff
@@ -1414,7 +1413,7 @@ bool System::command_info(const char * value, const int8_t id, JsonObject output
 #endif
     EMSESP::esp8266React.getNetworkSettingsService()->read([&](NetworkSettings & settings) {
         if (WiFi.status() == WL_CONNECTED && !settings.bssid.isEmpty()) {
-            node["BSSID"] = "set"; // TODO why is this not the actual value?
+            node["BSSID"] = "set"; // TODO why is this not the actual value??
```

Comment:
A while ago a user complains tht he does not want to publish his mac-address in the support info. So you have commented out all macs and ips and here we only see if a bssid is set or not.

Reply:
ok.